modules/shared/components/molecules/ImageCaption/ImageCaption.module.css

```diff
@@ -1,6 +1,11 @@
 .captionContent {
-  @apply flex justify-center items-center font-normal text-sm text-dark capitalize  px-xsvvl py-xxsv h-8 rounded-sm bg-white backdrop-blur-sm bg-opacity-64;
+  @apply flex justify-center items-center font-normal text-sm text-dark capitalize max-w-12xl px-xsvvl py-xxsv rounded-sm bg-white backdrop-blur-sm bg-opacity-64;
 }
```

Comment:
The class `max-w-12xl` throw an error in story book 

> (2:84) /Users/abdulrahman/mystories/pickify-frontend/modules/shared/components/molecules/ImageCaption/ImageCaption.module.css The `max-w-12xl` class does not exist, but `max-w-2xl` does. If you're sure that `max-w-12xl` exists, make sure that any `@import` statements are being properly processed before Tailwind CSS sees your CSS, as `@apply` can only be used for classes in the same CSS tree.

  1 | .captionContent {
> 2 |   @apply flex justify-center items-center font-normal text-sm text-dark capitalize max-w-12xl px-xsvvl py-xxsv rounded-sm bg-white backdrop-blur-sm bg-opacity-64;
    |                                                                                    ^
  3 | }